experimental/framework/tree-react-api/api-report/tree-react-api.alpha.api.md

```diff
@@ -5,15 +5,10 @@
 ```ts
 
 // @public
-export interface IReactTreeDataObject<TSchema extends ImplicitFieldSchema> extends ITreeDataObject<TSchema> {
-    readonly TreeViewComponent: (props: TreeViewProps<TSchema>) => React_2.JSX.Element;
-}
-
-// @public
-export interface ITreeDataObject<TSchema extends ImplicitFieldSchema> {
+export interface IReactTreeDataObject<TSchema extends ImplicitFieldSchema> {
     readonly config: TreeViewConfiguration<TSchema>;
-    readonly key: string;
```

Comment:
Since the tree is no longer being inserted into a directory, this `key` property is no longer needed.

Additionally, this base interface was not used outside of the derived `IReactTreeDataObject` interface, so I went ahead and collapsed the two together to simplify things.

Reply:
This distinction between interfaces was done so there could be a non-react dependent API for using shared tree data object kinds with built in schema in container schema.

Now that aquaduct can depend on tree, moving that logic there might make sense

Reply:
That was my assumption, but there isn't yet a clear need for such an interface outside of this, so I opted to remove it for now. Subsequent task(s) will include more data-object example work that I don't plan to leverage this library for. That will likely better inform if/where such an interface would be useful.